src/hooks/useHideEmojisByUniocode.ts

```diff
@@ -0,0 +1,6 @@
+import { useUnicodeToHide } from "../config/useConfig";
+
+export function useIsUnicodeHidden() {
+    const unicodeToHide = useUnicodeToHide();
+    return (emojiUnified: string) => unicodeToHide.includes(emojiUnified);
```

Comment:
When we switch unicodeToHide to a Set, we can use `has` for an O1 lookup. 
```suggestion
    return (emojiUnified: string) => unicodeToHide.has(emojiUnified);
```